### DISCUSSION TITLE
Internal Server Error

### DISCUSSION THREAD

**Discussion by ndlang** [Budibase/budibase#5484] [Help]
I got this notification where would I go to see what caused it:
![image](https://user-images.githubusercontent.com/17033357/163986709-33ede6ee-1f02-4d0b-bb30-fb69a216813b.png)

**Comment by melohagan** [comment]
Hi @ndlang 

Have a look at the console tab / network tab (Fn+F12). 

If this is a query datasource and the data is populating the table, there is a minor bug that's been identified, and there'll be a fix out for this with the next release.

**Comment by AssignGuard** [comment]
I am also getting a similar message that recently started appearing. I haven't added any new queries and the page that this error appears on seems to work fine. Has this been resolved?

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/106572626/172421579-a8c80850-52dd-43f8-8bac-2aadbbee6654.png">

<img width="718" alt="image" src="https://user-images.githubusercontent.com/106572626/172421704-e2ca095f-80ed-4b66-a947-f34de4cad055.png">

**Reply by melohagan** [reply]
Hi @AssignGuard 

There's another discussion here that's related: https://github.com/Budibase/budibase/discussions/6197

If you look at the **bbapps** logs do you see the same message? And as suggested, does removing and re-adding the datasource help? 

Are you using SQL Server as well?

**Reply by AssignGuard** [reply]
@melohagan if I re-add a datasource, does it persist my old queries? I'm afraid of losing all my queries.

**Reply by melohagan** [reply]
It would remove your queries unfortunately. Maybe you could try removing one of the troublesome queries and re-adding it? 
Although from the other discussion it did seem to be the datasource itself that was the issue. 

Export your app and import it as a new clone app. Then remove and re-add the datasource in the copy app. That way we can tell if that's the problem without deleting your queries.

**Reply by tailorck** [reply]
@melohagan - the app is too large to import it again 😢 
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/7613002/174840874-62f9d6ad-cfd7-40e8-b394-8395a10b876b.png">
